tests/__init__.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
```

Comment:
👀 This file should be empty. It just needs to be present for Python to see this folder as a package.